src/splitInteger.js

```diff
@@ -4,6 +4,7 @@
  *
  * @returns {number[]}
  */
+// eslint-disable-next-line strict
 function splitInteger(value, numberOfParts) {
```

Comment:
Consider adding a check to handle the case where `numberOfParts` is zero to avoid division by zero errors.